File: bundle/bundle.go
package main

import (
	"fmt"
	"io/ioutil"
	"os"
	"path/filepath"
)

func spriteBundle(dir string) {
	spritePath := filepath.Join(dir, "sprites")
	fmt.Println("sprites:", spritePath)

	spriteDir, err := ioutil.ReadDir(spritePath)
	if err != nil {
		panic(err)
	}

	for _, info := range spriteDir {
		name := info.Name()
		path := filepath.Join(spritePath, name)
		sheet := filepath.Join(dir, "public", "sprites", name)
		fmt.Println(name, "->", sheet)
		spritePacker(path, sheet)
	}
}

func main() {
	dir := os.Args[1]
	spriteBundle(dir)
}

File: bundle.js
const fs = require('fs')
const path = require('path')

const directory = 'sprites'

fs.readdir(directory, (error, files) => {
  files.forEach((name) => {
    let join = path.join(directory, name)
    let stat = fs.statSync(join)
    if (stat.isDirectory()) {
      console.log('is dir', join)
    } else if (stat.isFile()) {
      console.log('is file', join)
    }
  })
})
